package-lock.json

```diff
@@ -1231,29 +1156,15 @@
     "camelcase": {
       "version": "5.3.1",
       "resolved": "https://registry.npmjs.org/camelcase/-/camelcase-5.3.1.tgz",
-      "integrity": "sha512-L28STB170nwWS63UjtlEOE3dldQApaJXZkOI1uMFfzf3rRuPegHaHesyee+YxQ+W6SvRDQV6UrdOdRiR153wJg=="
-    },
-    "camelcase-keys": {
-      "version": "6.2.2",
-      "resolved": "https://registry.npmjs.org/camelcase-keys/-/camelcase-keys-6.2.2.tgz",
-      "integrity": "sha512-YrwaA0vEKazPBkn0ipTiMpSajYDSe+KjQfrjhcBMxJt/znbvlHd8Pw/Vamaz5EB4Wfhs3SUR3Z9mwRu/P3s3Yg==",
-      "requires": {
-        "camelcase": "^5.3.1",
-        "map-obj": "^4.0.0",
-        "quick-lru": "^4.0.1"
-      }
+      "integrity": "sha512-L28STB170nwWS63UjtlEOE3dldQApaJXZkOI1uMFfzf3rRuPegHaHesyee+YxQ+W6SvRDQV6UrdOdRiR153wJg==",
```

Comment:
+++ unchanged from previous version but now dev=true?

Reply:
actually looks like a lot of these are now dev=true but the package (version, sha) is unchanged from before.... 
🤔

Reply:
npm has had a history of different versions of the CLI producing slightly different `package-lock.json` files, so my initial instinct here was "🤷 , npm being npm?". However, then I actually went and did some research and learned something:

In this context, `dev: true` means "this package is either one of our `devDependencies` or is only depended on by our `devDependencies` ([source](https://docs.npmjs.com/cli/v6/configuring-npm/package-lock-json#dev)). This is a hint to npm itself--if you were to run `npm ci --production` it will not install anything in `package-lock.json` flagged as `dev`. It is _not_ saying "hey npm, it's fine for you to install a dev version of this package." That's why the integrity hash is unchanged--the same code is being installed as before. For our purposes, this doesn't mean much, since we install dev + non-dev dependencies in CI for our build.

`node-sass` had several dependencies in common with packages listed in our `devDependencies`. We removed `node-sass` from our `dependencies`, and npm recognized that those previously shared dependencies are now only required by our `devDependencies` and marked them as `dev: true` in `package-lock.json`.